frontend/packages/console-app/src/components/guided-tours/GuidedTourItem.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { CatalogTile } from '@patternfly/react-catalog-view-extension';
+import { GuidedTourItem, TourStatus } from './utils/guided-tour-typings';
+import TourItemHeader from './TourItemHeader';
+import TourItemDescription from './TourItemDescription';
+import TourItemFooter from './TourItemFooter';
+import './GuidedTourItem.scss';
+
+type GuidedTourItemProps = GuidedTourItem & TourStatus;
+
+const GuidedTourItem: React.FC<GuidedTourItemProps> = ({
+  iconURL,
+  altIcon,
+  name,
+  description,
+  status,
+  active,
+  duration,
+  prerequisites,
+}) => (
+  <CatalogTile
+    className="odc-guided-tour-item"
+    featured={active}
+    title={
+      <TourItemHeader
+        iconURL={iconURL}
```

Comment:
Why isn't the icon rendered as part of the catalog tile `icon` prop?

Reply:
The icon through prop is rendered in a separate div `card-head` than the title block which is in `card-header` and will not look inline as is given by UX without overriding CSS.
One way around it isto use `PF-Cards` , but again that will give a different outer look.

Reply:
Oh I see. UX isn't following the patternfly designs. They have created their own custom view.

@bmignano @beaumorley shouldn't we be trying to remain consistent with out views and follow the PF designs? We have other cataloges (operator hub, dev catalog) which use the same component and now guided tours has a different design.

Reply:
@christianvogt That's a good point. I think that there's been an effort to standardize catalog cards or tiles across PF so the design may have been created using a different set of standards than what is currently in OperatorHub. I agree that the tiles across the views you mentioned should be consistent and use the same view. Is it possible to show a screenshot of what it would look like using the same tile design?

Reply:
it would look something like this
![Screenshot from 2020-06-23 16-38-07](https://user-images.githubusercontent.com/24852534/85396837-1286a500-b570-11ea-9b44-d2a868112e18.png)